tox.ini

```diff
@@ -45,7 +45,7 @@ commands = flake8
 
 [testenv:py37-flake8]
 skip_install = true
-deps = flake8~=3.7,>=3.7.8
+deps = flake8~=5.0
```

Comment:
The version of flake being used was throwing an error when run. `5.0` appears to work for python 3.7